### PR TITLE
ARTEMIS-4404 Minor update to the artemis-docker build documentation

### PR DESCRIPTION
**Pull request by thatmikewilliams**
Clarification of steps required to create a docker image from ActiveMQ source
using the local-distribution method and a couple of formatting recommendations.

### CODE REVIEW

**Comment by clebertsuconic** [comment]
Can you amend your commit message to include the jira?


git commit --amend
git push origin -f <your-branch_name>

(that should update the commit)

File: artemis-docker/readme.md [hunk 1]
```diff
@@ -134,7 +148,7 @@ From within the `$ARTEMIS_DIST` folder:
 $ docker build -f ./docker/Dockerfile-ubuntu-11-jre -t artemis-ubuntu .
 ```
 
-# For Ubuntu (Build for linux ARMv7/ARM64)
+## For Ubuntu (Build for linux ARMv7/ARM64)
 ```
 $ docker buildx build --platform linux/arm64,linux/arm/v7 --push -t {your-repository}/apache-artemis:2.17.0-SNAPSHOT -f ./docker/Dockerfile-ubuntu-11 .
```
Comment (clebertsuconic) [inline artemis-docker/readme.md:153]: Were you able to make this work with 2.30?

**Comment by jbertram** [comment]
I fixed this with some git fu.

**Comment by jbertram** [comment]
Thanks for the contribution @thatmikewilliams!

File: artemis-docker/readme.md (hunk 1 above)
Comment (thatmaw) [reply artemis-docker/readme.md:153]: Hi Clebert. Sorry I missed the commit msg req, and thanks Justin for fixing it.
Yes I was able to get a cross platform version of 2.30.0 built. I updated karaf to 4.4.3 as per main which allowed the local build to complete successfully. The other steps worked as written in the doc.

**Pull request closed without merging**